compute_tools/src/compute.rs

```diff
@@ -390,6 +390,34 @@ impl ComputeNode {
         Ok(())
     }
 
+    // Gets the basebackup in a retry loop
+    #[instrument(skip_all, fields(%lsn))]
+    pub fn get_basebackup(&self, compute_state: &ComputeState, lsn: Lsn) -> Result<()> {
+        let mut timeout_ms = 500;
```

Comment:
timeout usually means "after this time, we will fail", rather than "we wait this long between attempts".  A better name might be something like `retry_period`

Reply:
Switched to `retry_period_ms`